reactor-core/src/main/java/reactor/core/Scannable.java

```diff
@@ -287,6 +287,11 @@ public boolean isScanAvailable() {
 			public String toString() {
 				return "UNAVAILABLE_SCAN";
 			}
+
+			@Override
+			public String stepName() {
+				return "UNAVAILABLE_SCAN";
```

Comment:
Don't we want step names to be camelCase or kebab-case?

Reply:
I think for these two it is fine or even better to have something slightly different, because it means the "operator" is not `Scannable`, so we won't be able to get back to parents further.